base/abstractarray.jl

```diff
@@ -2432,11 +2439,10 @@ function hash(A::AbstractArray, h::UInt)
     fibskip = prevfibskip = oneunit(linidx)
     first_linear = first(LinearIndices(linear_to_key))
     n = 0
-    while true
+    @inbounds while true
```

Comment:
This seems unlikely to matter, would be an unrelated change, and we typically consider this annotation to be undesirable unless strongly proven otherwise.
```suggestion
    while true
```